ledger/store/src/helpers/rocksdb/internal/id.rs

```diff
@@ -280,4 +274,5 @@ enum DataID {
     Test3,
     #[cfg(test)]
     Test4,
+    // NOTE: the last variant MUST NOT EXIST (at index = 255).
```

Comment:
the final variant is actually `65_535` right now, as `DataId` is represented with a `u16`; if we want to reduce it to `u8`, I'm all for it, as I doubt we will ever need more than 255 data types

Reply:
fixed in e22759f73